contracts/modules/TransferManager.sol

```diff
@@ -108,35 +103,36 @@ contract TransferManager is BaseModule, RelayerModule, OnlyOwnerModule, BaseTran
      * @param _wallet The target wallet.
      */
     function init(BaseWallet _wallet) public onlyWallet(_wallet) {
-        // setup static calls
-        _wallet.enableStaticCall(address(this), ERC721_ISVALIDSIGNATURE_BYTES);
-        _wallet.enableStaticCall(address(this), ERC721_ISVALIDSIGNATURE_BYTES32);
-        
+
         // setup default limit for new deployment
         if(address(oldLimitManager) == address(0)) {
             super.init(_wallet);
             return;
         }
         // get limit from previous LimitManager
-        uint256 currentLimit = oldLimitManager.getCurrentLimit(_wallet);
-        (uint256 pendingLimit, uint64 changeAfter) = oldLimitManager.getPendingLimit(_wallet);
+        uint256 current = oldLimitManager.getCurrentLimit(_wallet);
+        (uint256 pending, uint64 changeAfter) = oldLimitManager.getPendingLimit(_wallet);
         // setup default limit for new wallets
-        if(currentLimit == 0 && changeAfter == 0) {
+        if(current == 0 && changeAfter == 0) {
             super.init(_wallet);
             return;
         }
         // migrate existing limit for existing wallets
-        if(currentLimit == pendingLimit) {
-            limits[address(_wallet)].limit.current = uint128(currentLimit);
+        if(current == pending) {
+            limits[address(_wallet)].limit.current = uint128(current);
         }
         else {
-            limits[address(_wallet)].limit = Limit(uint128(currentLimit), uint128(pendingLimit), changeAfter);
+            limits[address(_wallet)].limit = Limit(uint128(current), uint128(pending), changeAfter);
         }
         // migrate daily pending if we are within a rolling period
         (uint256 unspent, uint64 periodEnd) = oldLimitManager.getDailyUnspent(_wallet);
         if(periodEnd > now) {
-            limits[address(_wallet)].dailySpent = DailySpent(uint128(currentLimit.sub(unspent)), periodEnd);
+            limits[address(_wallet)].dailySpent = DailySpent(uint128(current.sub(unspent)), periodEnd);
         }
+
+        // setup static calls
+        _wallet.enableStaticCall(address(this), ERC721_ISVALIDSIGNATURE_BYTES);
+        _wallet.enableStaticCall(address(this), ERC721_ISVALIDSIGNATURE_BYTES32);
```

Comment:
Why moving this down here? These lines won't be executed for new wallets or new deployments.

Reply:
Yes, I have revereted.